packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -186,6 +186,86 @@ export class PlaygroundWorkerEndpoint extends PHPWorker {
 	async replayFSJournal(events: FilesystemOperation[]) {
 		return replayFSJournal(this.__internal_getPHP()!, events);
 	}
+
+	async downloadWordPressAssets() {
+		await downloadWordPressAssets(this.__internal_getPHP()!);
+	}
+}
+
+async function downloadWordPressAssets(php: PHP) {
+	if (!php.requestHandler) {
+		logger.warn('No PHP request handler available');
+		return;
+	}
+
+	try {
+		const remoteAssetListPath = joinPaths(
+			php.requestHandler.documentRoot,
+			'wordpress-remote-asset-paths'
+		);
+		// If the remote asset list is not available this means that the WordPress static assets are already downloaded.
+		if (!php.fileExists(remoteAssetListPath)) {
+			return;
+		}
+		const wpVersion = await getLoadedWordPressVersion(php.requestHandler);
+		const staticAssetsDirectory =
+			wpVersionToStaticAssetsDirectory(wpVersion);
```

Comment:
@brandonpayton I keep getting confused by this name – it suggests every WordPress version has a static assets directory. How about renaming it to `staticAssetsUrlForMinifiedWPBuild`?

Reply:
Sorry for the confusion, @adamziel. The function only produces a directory name. What do you think about `staticAssetsSubdirForMinifiedWPBuild(wpVersion)` or a `staticAssetsUrlForMinifiedWPBuild(wpVersion, baseUrl)`?

Reply:
I like `staticAssetsUrlForMinifiedWPBuild` @brandonpayton !